blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+
+{% tabs %}
+{% highlight C# tabtitle=".NET 6 (~/Program.cs)" hl_lines="3 11" %}
+
+@using Syncfusion.Blazor.Popups
+
+// Add services to the container.
+
+builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddScoped<SfDialogService>();
+builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+await builder.Build().RunAsync();
+....
+
+{% endhighlight %}
+
+{% highlight c# tabtitle=".NET 5 and .NET 3.X (~/Program.cs)" hl_lines="1 10" %}
+
+@using Syncfusion.Blazor.Popups
+
+namespace WebApplication1
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            ....
+            builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+            builder.Services.AddScoped<SfDialogService>();
+            await builder.Build().RunAsync();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+open  **~/_MainLayout.razor** file and add SfDialogProvider
+
+{% tabs %}
+{% highlight razor tabtitle="~/_MainLayout.razor" %}
+
+<Syncfusion.Blazor.Popups.SfDialogProvider/>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Available Predefined dialogs
+
+There are three available predefined dialogs:
+
+        * Alert
+        * Confirm
+        * Prompt
+
+### Alert
+
+An [Alert](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_AlertAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) dialog box is used to display errors, warnings, and information that needs user awareness.
+
+Use the following code to render a simple alert dialog in an application.
+
+{% tabs %}
+{% highlight razor %}
+
+{% include_relative code-snippet/alert-dialog.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Alert Dialog](./images/blazor-alert-dialog.png)
+
+### Confirm
+
+A [Confirm](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_ConfirmAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) dialog displays a specified message along with ‘OK’ and ‘Cancel’ button. Used to get approval from user that appears before any critical action.
+
+Use the following code to render a simple confirm dialog in an application.
+
+{% tabs %}
+{% highlight razor %}
+
+{% include_relative code-snippet/confirm-dialog.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Confirm Dialog](./images/blazor-confirm-dialog.png)
+
+### Prompt
+
+A [Prompt](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_PromptAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) dialog is used to get input from the user. When the user enters "OK", the input value is returned. When they click "Cancel", the null value  returned.
```

Comment:
A [Prompt](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_PromptAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) dialog is used to get input from a user. When the user enters "OK", the input value is returned. When they click "Cancel", the null value is returned.